### PR TITLE
feat: add ListUnassignedExternalContact TransferExternalContact GetTransferExternalContactResult TransferGroupChatExternalContact

### PR DESCRIPTION
**Pull request by w3xse7en**
ListUnassignedExternalContact doc:https://open.work.weixin.qq.com/api/doc/90000/90135/92124
TransferExternalContact doc:https://open.work.weixin.qq.com/api/doc/90000/90135/92125
GetTransferExternalContactResult doc:https://open.work.weixin.qq.com/api/doc/90000/90135/92973
TransferGroupChatExternalContact doc:https://open.work.weixin.qq.com/api/doc/90000/90135/92127

### CODE REVIEW

File: external_contact.go [hunk 1]
```diff
@@ -81,3 +85,72 @@ func (c *WorkwxApp) MarkExternalContactTag(userID, externalUserID string, addTag
...
 }
+
+// ExternalContactUnassigned 离职成员的客户
+type ExternalContactUnassigned struct {
+	// HandoverUserID 离职成员的userid
+	HandoverUserID string `json:"handover_userid"`
+	// ExternalUserID 外部联系人userid
+	ExternalUserID string `json:"external_userid"`
+	// DimissionTime 成员离职时间
+	DimissionTime time.Time `json:"dimission_time"`
```
Comment (xen0n) [inline external_contact.go:96]: Apparently Tencent used archaic English here: [dimission](https://www.merriam-webster.com/dictionary/dimission) vs [demission](https://www.merriam-webster.com/dictionary/demission).

From quick Google searches "demission" has far greater number of hits (23300000 vs 384000), so what about replacing this particular word?

NOTE: Not doing so is also absolutely OK for me. Users of this library should be overwhelmingly Chinese and they really want to just search the Tencent name and expect to see results.

Comment (w3xse7en) [reply external_contact.go:96]: i think so too, and if Tencent spell wrong word, i will fix it as possible as i can like `StatusChangeEvent string xml:"StatuChangeEvent"`  `Notifier string xml:"Notifyer"`  and so on

**Comment by xen0n** [comment]
Looks very nice, thanks for the addition!

bors r+

**Comment by ocavue** [comment]
@xen0n Hi. Do you have any plan to release a new version/tag for these APIs. We would like to use some of the APIs in the latest code. It's better for the code maintainability if we don't need to use Git SHA as the version. Awsome project BTW, really appreciate your guys' work.

**Comment by xen0n** [comment]
@ocavue Sure, will do another alpha release soon. Expect `v1.0.0-alpha.2` later in the day (assuming China Standard Time).

**Comment by xen0n** [comment]
@ocavue `v1.0.0-alpha.2` is out!

**Comment by ocavue** [comment]
@xen0n Thanks!

**Pull request merged**